src/views/Header.js

```diff
@@ -71,6 +76,24 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     return this._getHeaderTitleString(lastScene);
   }
 
+  _getBackButtonTitleStyle(scene: NavigationScene): ?Style {
+    const lastScene = this._getLastScene(scene);
+    if (!lastScene) {
+      return null;
+    }
+    const {
+      headerBackTitleStyle,
```

Comment:
If you have a dark header in one screen and a light one in previous screen, it doesn't make much sense to inherit styles from previous screen. 

IMO each element in a screen should be themable by that screen only and not by other screens.